sdktests/server_side_events_alias.go

```diff
@@ -17,13 +17,11 @@ type aliasEventTestScenario struct {
 }
 
 func doServerSideAliasEventTests(t *ldtest.T) {
-	eventsConfig := baseEventsConfig()
-
 	userFactory := NewUserFactory("doServerSideAliasEventTests")
 
 	dataSource := NewSDKDataSource(t, mockld.EmptyServerSDKData())
 	events := NewSDKEventSink(t)
-	client := NewSDKClient(t, WithConfig(servicedef.SDKConfigParams{Events: &eventsConfig}), dataSource, events)
```

Comment:
This no longer needs to be specified, unless you're customizing event properties— if you include an `EventSink` then it will assume there should be at least a default event configuration.